backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -168,7 +168,7 @@ public void close() {
             new ImageFromDockerfile(imageName,doNotDeleteImageAfterUsage)
                 .withDockerfileFromBuilder(builder ->
                     additionalSteps.applyStep(builder
-                        .from("cassandra:3.11.15")
+                        .from("cassandra:4.0")
```

Comment:
4.1.latest please

Reply:
4.1 is GA Version, 4.0 is Stable version,
is it ok with 4.1?

Reply:
Yes